src/corner_kick/src/modules/Visualizer/components/Field.ts

```diff
@@ -0,0 +1,105 @@
+import * as PIXI from 'pixi.js';
+
+import { ROSConnector } from '~/ros/RosConnector';
+import { IFieldTopic } from '~/types/thunderbotsTopics';
+
+/**
+ * Displays a field in our visualizer
+ * Dimensions from the field are received from the ROS topic
+ * /backend/field
+ *
+ * TODO We are multiplying all values by 50 to make them bigger.
+ * Make this parameterizable and place it somewhere general.
+ *
+ * TODO Make field color parameterizable
+ * TODO Make field stroke width parameterizable
+ */
+export const Field = () => {
+  const field = new PIXI.Graphics();
+  const playingField = new PIXI.Graphics();
+
+  field.addChild(playingField);
+
+  // Susbscribe to the ROS topic /backend/field
+  ROSConnector.Instance.subscribeToTopic(
+    '/backend/field',
+    (topic: IFieldTopic) => {
+      // Clear both graphic containers as we will be redrawing them
+      field.clear();
+      playingField.clear();
+
+      field.lineStyle(1, 0xffffff);
+      playingField.lineStyle(1, 0xffffff);
```

Comment:
Color should be in constants file mentioned above

Reply:
Done